[BUG] Missing dev dependancy (aiohttp)

**Describe the bug**
aiohttp is a missing dev dependency in pyproject.toml

**To Reproduce**
Follow contributing.md 

1. **Fork and clone the repository**
   ```bash
   git clone https://github.com/yourusername/zenoo-rpc.git
   cd zenoo-rpc
   ```

2. **Create a virtual environment**
   ```bash
   python -m venv venv
   source venv/bin/activate  # On Windows: venv\Scripts\activate
   ```
>Note:
>The pip install doesn't work unless you do pip install -e ".[dev]" (at least for me). Tried it using Python v3.10 and v.3.13 

3. **Install development dependencies**
   ```bash
   pip install -e .[dev]
   ```
4. **Run all tests**
```sh
pytest
```

**Expected behavior**
pytest should run the tests successfully

**Error Message**

```
ImportError while loading conftest '/Users/.../tests/conftest.py'.
tests/conftest.py:4: in <module>
    import aiohttp
E   ModuleNotFoundError: No module named 'aiohttp'
```

**Environment (please complete the following information):**
 - OS: macOS 15.5
 - Python version: 3.10
 - Zenoo RPC version: 0.2.4

**Additional context**
Add any other context about the problem here.

**Checklist**
- [x] I have searched existing issues to ensure this is not a duplicate
- [x] I have provided a minimal code example that reproduces the issue
- [x] I have included the full error message and traceback
- [x] I have specified my environment details
- [x] I have checked the documentation and FAQ

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -49,6 +49,7 @@ dev = [
     "ruff>=0.1.0",
     "mypy>=1.0.0",
     "pre-commit>=3.0.0",
+    "aiohttp>=3.9.0",
 ]
 docs = [
     "mkdocs>=1.5.0",

